.github/workflows/nuget-publish.yml

```
@@ -2,7 +2,7 @@ name: Release Package Version
 on:
   push:
     tags:
-     - 2.*     
+     - 3.*     
```

```suggestion
     - *.*     
```

I wonder if we don't just make it valid for any tag?

---

Yeah, that should work.